### PR TITLE
Update wording for clarity

### PR DESCRIPTION
**Pull request by MaterArc**

### CODE REVIEW

File: features/project-miniplayer/data.json [hunk 1]
```diff
@@ -8,8 +8,8 @@
   "tags": ["New", "Featured"],
   "dynamic": true,
   "options": [
-    { "id": "position-right", "name": "Place the player to the right.", "type": 1 },
-    { "id": "position-bottom", "name": "Place the player to the bottom.", "type": 1 },
+    { "id": "position-right", "name": "Move the player to the right.", "type": 1 },
```
Comment (OneShot-Niko) [inline features/project-miniplayer/data.json:11]: Player is probably enough but
```suggestion
{ "id": "position-right", "name": "Move the project player to the right.", "type": 1 },
```

File: features/project-miniplayer/data.json [hunk 2]
```diff
@@ -8,8 +8,8 @@
...
+    { "id": "position-bottom", "name": "Move the player to the bottom.", "type": 1 },
```
Comment (OneShot-Niko) [inline features/project-miniplayer/data.json:12]:
```suggestion
    { "id": "position-bottom", "name": "Move the project player to the bottom.", "type": 1 },
```

**Review by OneShot-Niko — CHANGES_REQUESTED** [review]

**Comment by MaterArc** [comment]
@OneShot-Niko Thanks! That looks better...

**Comment by OneShot-Niko** [comment]
> @OneShot-Niko Thanks! That looks better...

Np!

**Review by rgantzos — APPROVED** [review]

**Pull request merged**